components/aws/sagemaker/train/src/train.py

```diff
@@ -26,11 +26,11 @@ def create_parser():
   parser.add_argument('--image', type=str, required=False, help='The registry path of the Docker image that contains the training algorithm.', default='')
   parser.add_argument('--algorithm_name', type=str, required=False, help='The name of the resource algorithm to use for the training job.', default='')
   parser.add_argument('--metric_definitions', type=_utils.yaml_or_json_str, required=False, help='The dictionary of name-regex pairs specify the metrics that the algorithm emits.', default={})
-  parser.add_argument('--training_input_mode', choices=['File', 'Pipe'], type=str, help='The input mode that the algorithm supports. File or Pipe.', default='File')
-  parser.add_argument('--hyperparameters', type=_utils.yaml_or_json_str, help='Dictionary of hyperparameters for the the algorithm.', default={})
+  parser.add_argument('--training_input_mode', type=str, required=False, choices=['File', 'Pipe'], help='The input mode that the algorithm supports. File or Pipe.', default='File')
+  parser.add_argument('--hyperparameters', type=_utils.yaml_or_json_str, required=False, help='Dictionary of hyperparameters for the the algorithm.', default={})
   parser.add_argument('--channels', type=_utils.yaml_or_json_str, required=True, help='A list of dicts specifying the input channels. Must have at least one.')
-  parser.add_argument('--instance_type', required=False, type=str, help='The ML compute instance type.', default='ml.m4.xlarge')
-  parser.add_argument('--instance_count', required=True, type=int, help='The registry path of the Docker image that contains the training algorithm.', default=1)
+  parser.add_argument('--instance_type', type=str, required=False, help='The ML compute instance type.', default='ml.m4.xlarge')
+  parser.add_argument('--instance_count', type=int, required=True, help='The registry path of the Docker image that contains the training algorithm.', default=1)
```

Comment:
Was this as a result of an auto-formatter? Seems odd to be messing with these in this PR.

Reply:
No, I added these myself it was just something I noticed. I can remove them if you think it's a bad idea.